### ISSUE TITLE
Using DGA4132 as repeater

### ISSUE THREAD

**Issue by SushiTee** [Ansuel/tch-nginx-gui#779]
Device Model: DGA4132 AGTHP 2.1.0
Gui Version: 9.3.93

Description of problem: I want to use the DGA4132 as repeater. Meaning there is the folowing setup:
- one hotspot with internet connection broadcasting it via wifi
- the DGA4132 without any connection to the internet

I now want the DGA4132 to connect to the hotspot via wifi and span it's own subnet in which other clients may connect via LAN or wifi using the internet of the wifi.

Basically a repeater with it's own subnet.

I installed the luci GUI and there is a subpage to join wifi networks but it doesn't work (it's just loading all the time). I guess it is because all the _iw_ and _iwlist_ stuff does not work with the DGA4132.

Is there any way to achieve that? Best would be an additional mode for the internet connection in the GUI.

**Comment by FrancYescO** [comment]
same https://github.com/Ansuel/tch-nginx-gui/issues/777
